.ci.yaml

```diff
@@ -0,0 +1,32 @@
+# Describes the targets run in continuous integration environment.
```

Comment:
1. does this need a license comment?
2. I'd prefer a link to a doc in the cocoon source tree (or anywhere, as long as there is only one). this schema is bound to change :)

Reply:
1. I looked at cirrus and devicelab yaml files and didn't see license headers, so I didn't include one here
2. That's a good point (make it easier to copy this config to other repos). For now, is it okay to leave this in and later this week I can add documentation in Cocoon on the configs?

Reply:
SGTM